java/src/main/java/io/opentracing/contrib/ClientTracingInterceptor.java

```diff
@@ -299,7 +299,7 @@ public ClientTracingInterceptor build() {
         }
     }
 
-    enum ClientRequestAttribute {
+    public enum ClientRequestAttribute {
```

Comment:
You don't need to make this change, but FYI I regret this approach... would be less code and more flexibility to do a `SpanDecorator` here like we did in Go. https://github.com/grpc-ecosystem/grpc-opentracing/blob/master/go/otgrpc/options.go#L41

In case you feel inspired, we could add something similar as an alternative.